hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -78,8 +78,18 @@ def error500(e):
 
 @app.get("/")
 def root():
+    """
+    root endpoint to make sure the API is live and check when it was last updated
+    """
+    context = bottle.request.environ.get("apig_wsgi.context")
+    invoked_function_arn = context.invoked_function_arn
+    client = boto3.client("lambda")
+    last_modified = client.get_function_configuration(
+        FunctionName=invoked_function_arn
+    )["LastModified"]
+
     return {
-        "message": "Hello World, HMA",
+        "message": f"Welcome to the HMA API! Last Modified/Deployed: {last_modified}",
```

Comment:
Can you add last modified to a separate field in the response? That way, we can automate easily. I was planning on scheduling an alarm for when the Continuous deploy script fails.